phoenix5-spark/README.md

```diff
@@ -65,21 +77,67 @@ public class PhoenixSparkRead {
     public static void main() throws Exception {
         SparkConf sparkConf = new SparkConf().setMaster("local").setAppName("phoenix-test")
             .set("spark.hadoopRDD.ignoreEmptySplits", "false");
-        JavaSparkContext jsc = new JavaSparkContext(sparkConf);
-        SQLContext sqlContext = new SQLContext(jsc);
+        SparkSessinon spark = SparkSession.builder().config(sparkConf).getOrCreate();
         
         // Load data from TABLE1
-        Dataset<Row> df = sqlContext
+        Dataset<Row> df = spark
             .read()
             .format("phoenix")
             .option("table", "TABLE1")
+            .option("jdbcUrl", "jdbc:phoenix:zkHost:zkport")    
```

Comment:
see above, this applies to all uses with explicit jdbcUrl

Reply:
Done